formula-android/src/main/java/com/instacart/formula/android/FragmentStore.kt

```diff
@@ -70,13 +71,34 @@ class FragmentStore @PublishedApi internal constructor(
         val EMPTY = Builder().build {  }
     }
 
+    private val features = mutableMapOf<FragmentId, FeatureEvent>()
```

Comment:
Keeps track of initialized features. Makes it easier to implement `getViewFactory`

Reply:
Doesn't need to be thread safe, right?

Reply:
No, since it's updated via fragment lifecycle events that come on the main thread.